### DISCUSSION TITLE
Looking for some localization strings

### DISCUSSION THREAD

**Discussion by tadeu2** [OrcaSlicer/OrcaSlicer#1484] [Q&A]
@SoftFever where i can find the localization for the strings whose aren´t located in the po file? Such as index.html. I have translated the spanish po file bu those strings arent translated.

Thanks a lot!

**Comment by SoftFever** [comment]
Hi @tadeu2 
You can find these strings here: https://github.com/SoftFever/OrcaSlicer/blob/main/resources/web/data/text.js

**Reply by tadeu2** [reply]
Hi @SoftFever, is possible localizate it in several languages? Like gettext.

**Reply by SoftFever** [reply]
Yes, it is possible. If you open that js file, you can find other languages in it

**Comment by tadeu2** [comment]
Ok, i ve closed pull request until i finish that file
